<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_chart_builderClass</class>
 <widget class="QMainWindow" name="qt_chart_builderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_chart_builder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <widget class="QPushButton" name="btnAdd">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="editX">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="editY">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRemove">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnUpdate">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableData"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_chart_builder.qrc"/>
 </resources>
 <connections/>
</ui>
